runtimes/core/src/metrics/exporter/mod.rs

```diff
@@ -0,0 +1,7 @@
+mod gcp;
+pub use gcp::Gcp;
+
+#[async_trait::async_trait]
```

Comment:
Do we still need this? I think newer versions of Rust supported async traits natively for some use cases